scene/2d/gpu_particles_2d.cpp

```diff
@@ -910,10 +910,14 @@ GPUParticles2D::GPUParticles2D() {
 	RS::get_singleton()->particles_set_draw_pass_mesh(particles, 0, mesh);
 
 	one_shot = false; // Needed so that set_emitting doesn't access uninitialized values
+	// This is a bit tricky but we have to set only the seed here and not uset set_seed()
+	// Otherwise particles won't randomize on their first appearance and will all look the same.
+	// We don't want to use set_seed(randf) because that will affect the docs.
```

Comment:
You can use `ADD_PROPERTY_DEFAULT` for the docs to force a fixed value (e.g. 0) and avoid having to do hacks.

Reply:
Done!